test/_data/flutter_style_todos/a.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// test w/ `pub run test -N flutter_style_todos`
+
+// BAD
+//TODO
+// TODO
+// TODO: bla
+// TODO(user) remove
+// TODO:(user): Button
+// TODO(user) : We
+/// nothing here
+/// TODO(user): We
+// TODO(#12357): Bad username
+
+// GOOD
+// TODO(user): We
+// TODO(user-name): We
+// TODO(user1,user2): We
```

Comment:
eh, no need to support commas here i think

Reply:
I added that because of:
https://github.com/flutter/flutter/blob/7598050fff00f9b15b08c50d8a3e2f613851874e/packages/flutter_tools/bin/fuchsia_tester.dart#L94

Reply:
we should just change that to whoever last edited the line.